CONTRIBUTING.md

```diff
@@ -37,3 +37,30 @@ Contributions made by corporations are covered by a different agreement than the
  
 3) Add or change sample Gist IDs to the [index file](https://github.com/dart-lang/dart-pad/blob/master/web/index.html#L54),
    and submit a PR for review.
+
+## How to run DartPad locally
+
+* To run the DartPad against the regular serving backend:  `grind serve`.  This serves the
```

Comment:
I crashed because I don't have `vulcanize` installed? Do we need to list dependencies?

Reply:
looks like we need node v6, too?

Reply:
We require node for vulcanize, yes.

Reply:
Put that up top? Tell folks they need vulcanize + node 6 installed.

Reply:
npm v5 is actually fine too.  Will mention this.